src/widgets/Forms/AddExpenseForm/AddPaymentSourceForm.tsx

```diff
@@ -1,21 +1,23 @@
 import FormWrapper from 'src/widgets/Forms/FormWrapper.tsx';
-import { Button, FormControl, FormGroup, InputLabel, OutlinedInput, Stack, Typography } from '@mui/material';
+import { Button, FormControl, FormGroup, InputLabel, OutlinedInput, Stack, Typography, useTheme } from '@mui/material';
 import { useMutation } from '@tanstack/react-query';
 import { createPaymentSource, TCreatePaymentSourceInput, TPaymentSource } from 'src/shared/api/paymentsSourceApi.ts';
 import { useEffect, useState } from 'react';
 import { toast } from 'react-toastify';
-import { matchIsValidColor, MuiColorInput } from 'mui-color-input';
 import { TErrorResponse } from 'src/shared/api/rootApi.ts';
 import handleError from 'src/utils/errorHandler.ts';
 import useLoadPaymentSources from 'src/entities/paymentSource/hooks/useLoadPaymentSources.ts';
+import { Colorful } from '@uiw/react-color';
 
 type TAddPaymentSourceFormProps = {
   closeModal: () => void;
 };
 
 const AddPaymentSourceForm = ({ closeModal }: TAddPaymentSourceFormProps) => {
   const [title, setTitle] = useState('');
-  const [color, setColor] = useState('#ffffff');
+  //random color always 6 symbols
+  const [color, setColor] = useState(`#${Math.floor(Math.random() * 16777215).toString(16)}`);
```

Comment:
and then the magic...

Reply:
Hmmm.. feels like duplicate code... your right.